frontend/src/app/components/containers/complaints/outcomes/hwcr-equipment/equipment-item.tsx

```diff
@@ -139,7 +139,7 @@ export const EquipmentItem: FC<EquipmentItemProps> = ({ equipment, isEditDisable
             <div>
               <dt>Latitude/Longitude</dt>
               <dd>
-                {equipment.yCoordinate} {equipment.xCoordinate}
+                {equipment.yCoordinate}, {equipment.xCoordinate}
```

Comment:
This will cause a comma to appear on the screen when there are no coordinates.  Also need to wrap this in a conditional statement to verify coordinates exist before rendering.

![image](https://github.com/bcgov/nr-compliance-enforcement/assets/19497416/e535eba4-acc5-4080-acf6-33ca5de47970)